doc/source/user_guide/beginners_guide.rst

```diff
@@ -0,0 +1,345 @@
+            self.weights2 = nn.Parameter(self.scale * torch.rand(in_channels, out_channels, self.modes1, self.modes2, dtype=torch.cfloat))
+
+        def forward(self, x):
+            batchsize = x.shape[0]
+            # Compute Fourier coeffcients up to factor of e^(- something constant)
+            x_ft = torch.fft.rfft2(x)
+
+            # Multiply relevant Fourier modes
+            out_ft = torch.zeros(batchsize, self.out_channels, x.size(-2), x.size(-1)//2 + 1, dtype=torch.cfloat, device=x.device)
+            out_ft[:, :, :self.modes1, :self.modes2] = self.compl_mul2d(x_ft[:, :, :self.modes1, :self.modes2], self.weights1)
+            out_ft[:, :, -self.modes1:, :self.modes2] = self.compl_mul2d(x_ft[:, :, -self.modes1:, :self.modes2], self.weights2)
+
+            # Return to physical space
+            x = torch.fft.irfft2(out_ft, s=(x.size(-2), x.size(-1)))
+            return x
+
+b) MLP Layer
+~~~~~~~~~~~~
+
+After the spectral layer, an MLP (Multi-Layer Perceptron) layer is applied. This introduces non-linearity and helps in learning complex patterns.
+
+Function:
+
+- Applies non-linear transformations to the output of the spectral layer
+- Helps in learning more complex mappings
+
+Implementation:
+
+.. code-block:: python
+
+    self.mlp = nn.Sequential(
+        nn.Linear(self.hidden_channels, self.hidden_channels),
+        nn.GELU(),
+        nn.Linear(self.hidden_channels, self.hidden_channels),
+    )
+
+3. Output Projection Layer
+--------------------------
+
+The final layer projects the output back to the desired output dimension.
+
+Function:
+
+- Maps the high-dimensional representation back to the target output dimension
+
+Implementation:
+
+.. code-block:: python
+
+    self.Q = nn.Linear(self.hidden_channels, self.out_channels)
+
+Putting It All Together
+-----------------------
+
+The FNO model combines these components in a sequential manner:
+
+1. The input data is first lifted to a higher dimension by the data projection layer.
+2. It then passes through multiple FNO blocks, each consisting of a spectral layer and an MLP layer.
+3. Finally, the output projection layer maps the result back to the desired output dimension.
+
+Here's a simplified version of how these components come together in the forward pass:
+
+.. code-block:: python
+
+    def forward(self, x):
+        x = self.fc0(x)  # Lifting layer
+        for i in range(self.num_layers - 1):
+            x1 = self.conv_layers[i](x)
+            x2 = self.w_layers[i](x)
+            x = x1 + x2
+            x = self.mlp_layers[i](x)
+        x = self.fc1(x)  # Output projection
+        return x
+
+This structure allows FNOs to efficiently learn mappings between function spaces, making them particularly effective for solving PDEs and other complex mathematical problems. Now finally let's see how to train an FNO model.
+
+3. Set Up the FNO Model
```

Comment:
Not sure we need another full training guide within the docs here

Reply:
Its more of just a simplified version and how to train it.